.github/workflows/genai_package.yml

```diff
@@ -0,0 +1,47 @@
+name: genai_package
+on: pull_request
+jobs:
+  ubuntu_genai_package:
+    if: false
+    runs-on: ubuntu-20.04
+    steps:
+      - uses: actions/checkout@v4
+        with:
+          submodules: recursive
+      - uses: actions/setup-python@v4
+        with:
+          python-version: 3.8
+      - run: mkdir ./ov/
+      - run: curl https://storage.openvinotoolkit.org/repositories/openvino/packages/2024.1/linux/l_openvino_toolkit_ubuntu20_2024.1.0.15008.f4afc983258_x86_64.tgz | tar --directory ./ov/ --strip-components 1 -xz
+      - run: sudo ./ov/install_dependencies/install_openvino_dependencies.sh
+      - run: source ./ov/setupvars.sh && cmake -DCMAKE_BUILD_TYPE=Release -S ./ -B ./build/
+      - run: source ./ov/setupvars.sh && cmake --build ./build/ --config Release --target package -j
+      - run: source ./ov/setupvars.sh && cmake --install ./build/ --config Release --prefix ov
+      - run: ov/samples/cpp/build_samples.sh -b "${{ github.workspace }}/s pace"
+      - run: source ./ov/setupvars.sh && python -m pip install --upgrade-strategy eager -r text_generation/causal_lm/cpp/requirements.txt
+      - run: source ./ov/setupvars.sh && python -m pip install ./thirdparty/openvino_tokenizers/[transformers]
+      - run: source ./ov/setupvars.sh && optimum-cli export openvino --trust-remote-code --weight-format fp16 --model TinyLlama/TinyLlama-1.1B-Chat-v1.0 TinyLlama-1.1B-Chat-v1.0
+      - run: source ./ov/setupvars.sh && timeout 50s "${{ github.workspace }}/s pace/intel64/Release/greedy_causal_lm" .\TinyLlama-1.1B-Chat-v1.0\ ""
```

Comment:
My approach addresses https://github.com/openvinotoolkit/openvino.genai/pull/334#discussion_r1605687525. Moreover it's easier to see which command failed. The drawback is that introduces a long prefix `call w_openvino_toolkit_windows_2024.2.0.dev20240515_x86_64\setupvars.bat &&`. This can be addressed later by hardcoding env vars at a job level which would setupvars set or introduce an alias or a file link to reduce the length of the prefix.